packages/actor-tests/bedrock/deposit-eth.ts

```diff
@@ -0,0 +1,88 @@
+import * as fs from 'fs'
+
+import { utils, Wallet, providers, constants } from 'ethers'
+import {
+  CrossChainMessenger,
+  ETHBridgeAdapter,
+  StandardBridgeAdapter,
+} from '@eth-optimism/sdk'
+import { predeploys } from '@eth-optimism/contracts-bedrock'
+import { sleep } from '@eth-optimism/core-utils'
+
+import { actor, setupActor, run } from '../lib/convenience'
+
+interface Context {
+  wallet: Wallet
+}
+
+actor('Dev account sender', () => {
```

Comment:
I think that this is good for now, we could technically incorporate `hardhat-deploy` and use a network to pull in deployment artifacts if we really wanted to, although that adds a lot of overhead so just using a path to a json file is more simple

Reply:
Yeah, I want to avoid incorporating Hardhat to keep thing simple.